### ISSUE TITLE
the split of dataset

### ISSUE THREAD

**Issue by Zizzzzzzz** [abhishek1411/event-rgb-fusion#6]
In paper, the split of dataset is 30 sequences for training, 3 sequences validation, and 8 sequences testing.  But in DSEC_detection_labels folder, the split of dataset is 33 sequences for training, 3 sequences validation, and 5 sequences testing. I am confused.

**Comment by abhishek1411** [comment]
The 8 sequences for testing in the paper were divided into 5 sequences for testing and 3 sequences for validation when the labels were released publicly. You could use the entire 8 sequences for testing and split a part of the training set into validation for hyperparameter tuning.

**Comment by Zizzzzzzz** [comment]
OK. Thank you!
